### DISCUSSION TITLE
Affichage conditionnel d'un composant

### DISCUSSION THREAD

**Discussion by anistouati** [MTESSDev/FRW#281] [Annonces]
Bonjour,

Je n'arrive pas a trouver la syntaxe pour utiliser un v-if quand le nom de composant contient un point "."
Ca fonctionne bien quand le nom est un mot, ou quand c'est un mot avec un underscode "_"

Exemple : 
`v-if: val('calendrier.combien_personne') === 'AUCUNE'`

**Comment by j0ber** [comment]
Le nom de composant ne doit pas contenir de point. Le point est réservé pour une utilisation avec les groupes ex. NomGroupe. NomChamp.
